src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,54 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.util.Arrays;
+import java.util.Map;
```

Comment:
Don’t use two-dimension arrays and HashMap(or any other Map). The task specifies not to use Map data structures. Consider implementing the functionality without using Maps.